src/hooks/productData/posthog_ai.tsx

```diff
@@ -413,16 +414,50 @@ export const posthog_ai = {
             ],
             skills: [
                 {
-                    name: 'Query cupid',
-                    description: 'Joins tables across tools like they were meant to be together',
+                    name: 'Fixing SQL mistakes',
+                    description: 'Fixes your errors to make sure you get with the data you want and need',
+                    sticker: <StickerPath className="size-6" />,
+                    percent: 50,
+                },
+                {
+                    name: 'Query autocomplete',
+                    description: 'Autocompletes your code, so that you have time for the important stuff',
+                    sticker: <StickerPath className="size-6" />,
+                    percent: 0,
+                },
+            ],
+        },
+        {
+            title: 'Data Pipelines',
+            headline: 'Data Pipelines',
+            layout: 'ai',
+            icon: <IconPlug className="size-5" />,
+            color: 'blue',
+            description:
+                'Transform data into PostHog and export your data out of PostHog to your favorite destinations.',
+            images: [
+                {
+                    src: 'https://res.cloudinary.com/dmukukwp6/image/upload/v1730806654/data_warehouse_99ac92f444.png',
+                    alt: 'Data Warehouse',
+                    className: 'max-h-64',
+                    // stylize: true,
+                    // shadow: true,
+                },
+            ],
+            skills: [
+                {
+                    name: 'Destinations set up',
+                    description:
+                        'Quickly set up destinations. Write custom (hog) code and automatically filter on the right events',
```

Comment:
Both of these first sentences in the descriptions are duplicative of the titles, though the descriptions are better since they're actions instead of phrases. But maybe more like: "Connect & configure new destinations" etc